examples/common/python/shared_kv/remote_lmdb/lmdb_request_handler.py

```diff
@@ -0,0 +1,258 @@
+                    response = "v\n" + escape(result)
+                else:
+                    response = "n"
+            else:
+                logger.error("Invalid args for cmd Get")
+                response = "e\nInvalid args for cmd Get"   
+
+        #Set
+        elif (cmd=="S"):
+            if len(args) == 4:
+                result = self.set(args[1], args[2], args[3])
+                if result:
+                    response = "t"
+                else:
+                    response = "f" 
+            else:
+                logger.error("Invalid args for cmd Set")
+                response = "e\nInvalid args for cmd Set"  
+
+        #Remove
+        elif (cmd=="R"):
+            if len(args) == 3 or len(args) == 4:
+                if len(args) == 3:
+                    result = self.remove(args[1], args[2])
+                else:
+                    result = self.remove(args[1], args[2], value=args[3])
+                if result:
+                    response = "t"
+                else:
+                    response = "f"
+            else:
+                logger.error("Invalid args for cmd Remove")
+                response = "e\nInvalid args for cmd Remove"  
+
+        else:
+            logger.error("Unknown cmd")
+            response = "e\nUnknown cmd"
+        return response
+
+    def render_GET(self, request):
+        response = 'Only POST request is supported'
+        logger.error("GET request is not supported." + \
+            " Only POST request is supported")
+        
+        return response
+
+    def render_POST(self, request):
+        response = ""
+       
+        logger.info('Received a new request from the client')
+
+        try :
+            # Process the message encoding
+            encoding = request.getHeader('Content-Type')
+            data = request.content.read().decode('utf-8')
+
+            if encoding == 'text/plain; charset=utf-8' :
+                response = self._process_request(data)
+            else :
+                response = 'UNKNOWN_ERROR: unknown message encoding'
+                return response
+
+        except :
+            logger.exception('exception while decoding http request %s', 
+                request.path)
+            response = 'UNKNOWN_ERROR: unable to decode incoming request '
+            return response
+
+        # Send back the results
+        try :
+            logger.info('response[%s]: %s', encoding, response.encode('utf-8'))
+            request.setHeader('content-type', encoding)
+            request.setResponseCode(http.OK)
+            return response.encode('utf-8')
+
+        except :
+            logger.exception('unknown exception while processing request %s', 
+                request.path)
+            response = 'UNKNOWN_ERROR: unknown exception processing ' + \
+                'http request {0}'.format(request.path)
+            return response
+
+#------------------------------------------------------------------------------
```

Comment:
Below functions open/close/set/get/remove/lookup are same as in shared_kv_interface.py. Could you directly use SharedKV class instead of duplicating the function definition ?

Reply:
Done.